pkgs/development/python-modules/penelope/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+}:
+buildPythonPackage rec {
+  name = "penelope";
+  version = "0.9.2";
+
+  src = fetchFromGitHub {
+    owner = "brightio";
+    repo = "penelope";
+    rev = "v${version}";
+    hash = "sha256-KlbB7i009UttTp+/MaF3y9CGUkAGEQQq/W+hQYKk9CY=";
+  };
+
+  preBuild = ''
+    cat > setup.py << EOF
+    from setuptools import setup
+
+    setup(
+      name='penelope',
+      version='${version}',
+      scripts=[
+        'penelope.py',
+      ],
+    )
+    EOF
+  '';
```

Comment:
Please submit this as a PR upstream and then fetch the commit using `fetchpatch`. Also please consider using `pyproject.toml` (see PEP 517) instead of `setup.py`. If you do so then `format = "pyproject";` or `pyproject = true;` will be needed,

Reply:
A major argument is that `pip install .` will _just work_ ;)

Reply:
done :+1: